src/api/libcellml/generator.h

```diff
@@ -95,7 +95,7 @@ class LIBCELLML_EXPORT Generator
      *
      * @return The interface code as a @c std::string.
      */
-    std::string interfaceCode() const;
+    std::string interfaceCode();
```

Comment:
I think this method and `implementationCode` should retain their constness.

Reply:
I completely agree! Done in 81bc8f5d.